src/functionaltests/java/com/ericsson/ei/subscriptions/trigger/SubscriptionTriggerSteps.java

```diff
@@ -214,6 +224,35 @@ private void validateSubscriptionsSuccessfullyAdded(String endPoint) throws Exce
         }
     }
 
+    /**
+     * Checks that an enpoint got at least the number of calls as expected.
+     * 
+     * @param endpoints
+     *            List of endpoints to check.
+     * @param expectedCalls
+     *            Integer with the least number of calls.
+     * @return true if all endpoints had atleast the number of calls as
+     *         expected.
+     * @throws JSONException
+     * @throws InterruptedException
+     */
+    private boolean allEndpointsGotAtLeastOneCall(final List<String> endpoints, int expectedCalls)
+            throws JSONException, InterruptedException {
+        List<String> endpointsToCheck = new ArrayList<String>(endpoints);
```

Comment:
remove String it's diamonds type   
List< String > endpointsToCheck = new ArrayList<>(endpoints);

Reply:
Done.